contract/contracts/PixelDevsUkraineDonation.sol

```diff
@@ -80,11 +83,16 @@ contract PixelDevsUkraineDonation is
     }
 
     function withdraw() public onlyOwner {
-        payable(msg.sender).transfer(address(this).balance);
+        payable(withdrawWallet).transfer(address(this).balance);
     }
 
     function switchContractState() public onlyOwner {
         contractState = !contractState;
         emit ContractStateUpdated(!contractState, contractState);
     }
+
+    function setWithdrawWallet(address _withdrawWallet) public onlyOwner {
+        emit WithdrawWalletUpdated(withdrawWallet, _withdrawWallet);
+        withdrawWallet = _withdrawWallet;
+    }
```

Comment:
When we are hard coding the Ukraine DAO treasury address I think it's fair to also remove this